requirements.txt

```diff
@@ -7,4 +7,7 @@ pefile == 2023.2.7
 semantic_version == 2.10.0
 GitPython == 3.1.31
 pyyaml == 6.0.0
-cryptography == 40.0.1
\ No newline at end of file
+cryptography == 40.0.1
+openpyxl == 3.1.2
+XlsxWriter == 3.0.9
+pywin32 == 305; sys_platform == 'win32'
```

Comment:
nit: use 306 here. dependabot will just immediately yell at me. 305 in the setup.py is perfectly fine however as you are supposed to put the minimum viable version in the setup.py so that there is less chance of conflicts when people install multiple modules.